test/sequential/test-dgram-pingpong.js

```diff
@@ -32,47 +21,42 @@ function pingPongTest(port, host) {
   });
 
   server.on('listening', function() {
-    console.log('server listening on ' + port + ' ' + host);
+    console.log('server listening on ' + port);
 
-    const buf = new Buffer('PING');
     const client = dgram.createSocket('udp4');
 
-    client.on('message', function(msg, rinfo) {
-      if (debug) console.log('client got: ' + msg +
-                             ' from ' + rinfo.address + ':' + rinfo.port);
-      assert.equal('PONG', msg.toString('ascii'));
-
-      count += 1;
+    client.on('message', function(msg) {
```

Comment:
I could go either way on it. I left it without `common.mustCall()` and used the `assert.strictEqual(pongsReceived, N);` instead so that it would be readily apparent that it's the analogous check to `assert(pingsReceived >= N);`.